src/js/components/Diagram/stories/Progressing.js

```diff
@@ -51,14 +51,22 @@ export const Progressing = () => {
 
   if (topRow.length >= 3) {
     connections.push(
-      connection('3', '5', { anchor: 'horizontal', color: 'brand' }),
+      connection('3', '5', {
+        anchor: 'horizontal',
+        animation: { type: 'pulse', duration: 500, size: 'small' },
+        color: 'brand',
+      }),
     );
   }
 
   return (
     <Grommet theme={grommet}>
       <Box align="start" pad="large">
-        <Text> Adding and removing nodes</Text>
+        <Text>
+          {' '}
```

Comment:
```suggestion
```

Reply:
Must've bugged this one in the linter check somehow but this got sorted out now.